### PR TITLE
Add system overview view

### PR DESCRIPTION
**Pull request by dixsie**
- Sets up system overview layout with out of the box Bootstrap
- Creates overview quick links and overview events
- Adds date-fns for date/time conversions

Signed-off-by: Dixsie Wolmers <dixsiew@gmail.com>

### CODE REVIEW

**Comment by dixsie** [comment]
Closes #4

File: src/views/Overview/Home.vue [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+            <b-col sm="6">
+              <dl>
+                <dt>SERIAL NUMBER</dt>
+                <dd>{{ system.SerialNumber || "N/A" }}</dd>
+              </dl>
+            </b-col>
+            <b-col sm="6">
+              <dl>
+                <dt>FIRMWARE VERSION</dt>
+                <dd>{{ software.Version || N/A }}</dd>
```
Comment (yoshiemuranaka) [inline src/views/Overview/Home.vue]: Does "N/A" need to be wrapped in quotes?

**Comment by yoshiemuranaka** [comment]
Also maybe update commit message, Moment to date-fns

File: src/views/Overview/Home.vue (hunk 1 above)
Comment (dixsie) [reply src/views/Overview/Home.vue]: without quotes it shows "NaN" in the view

Comment (dixsie) [reply src/views/Overview/Home.vue]: done

**Review by yoshiemuranaka — APPROVED** [review]

**Review by derick-montague — APPROVED** [review]

**Comment by derick-montague** [comment]
Integrated into webui-vue in https://gerrit.openbmc-project.xyz/c/openbmc/webui-vue/+/28603

**Pull request merged**